src/gulp-tasks-pack.js

```diff
@@ -36,6 +36,14 @@ export function packTasks (gulp) {
           if (err) {
             throw err;
           }
+
+          const newTarballName = `${json.name}-${json.version}-src-with-dependencies.tgz`;
```

Comment:
I may be missing something, but any reason why not setting the tarballName to include `src-with-dependencies`? It seems to me the rename is not necessary, look at line 34.

https://github.com/grommet/grommet-toolbox/blob/bbfff06a2e03dba755250e9dc5cb468be70498a8/src/gulp-tasks-pack.js#L34